scripts/local_integration_test.sh

```diff
@@ -44,48 +90,66 @@ pushd $deployDir
 export STAGING_IMAGE=$imageUnderTest
 envsubst < Dockerfile.in > Dockerfile
 echo "Building app container..."
-docker build -t $APP_IMAGE . || gcloud docker -- build -t $APP_IMAGE .
+docker build -t $APP_IMAGE . || gcloud docker -- build -t $APP_IMAGE . || usage "Error building test-app image from base image \"${imageUnderTest}\", please make sure it exists!"
+
+
+if [[ "$WITH_GCP" ]]; then
```

Comment:
I don't think it makes sense to me to add these integration tests to `the local_integration_test.sh` script. I think we originally added this script in order to test things that were outside the scope of the runtimes_common integraion testing framework (namely, container shutdown hooks).

I think if we wanted to make it easier to run the runtimes_common tests, a better way to do it would be to just simulate a cloudbuild execution locally, using the `container-builder-local` emulator that was just released as part of the Cloud SDK (see [release notes](https://cloud.google.com/sdk/docs/release-notes#google_cloud_container_builder)).